_config.yml

```diff
@@ -76,16 +79,8 @@ links:
     url: /enforcement
   - title: Resource Channels
     url: /resource-channels
-  - title: RFC Process
-    url: /rfc-process
-  - title: Signup Policy
-    url: /rfc-signup
-  - title: Slackbot Language Shorthands
-    url: /rfc-slackbot-language-shorthands
-  - title: Pairing & Review
-    url: /rfc-pairing
-  - title: Docents
-    url: /rfc-docents
+  - title: Policies
+    url: /rfc-policies
```

Comment:
So, thinking about this a bit more... I wish we had a better model for dealing with the Slack Post <--> Website documentation thing. Maybe we should start using github as the central place for authoring and maintaining RFCs, instead of Slack? That would let us actually use Github's review functionality and make it more obvious for a lot of people how we can add more stuff. Thoughts? Slack posts are kinda janky anyway :\